### DISCUSSION TITLE
I'd love the ability to enable ElevenClock on a per-monitor basis

### DISCUSSION THREAD

**Discussion by aydenking** [martinet101/ElevenClock#386] [General]
Hello,

I have three monitors and a tv connected to my pc. I'd like to hide the clock on just the tv's taskbar, but still show it on the taskbar of my two secondary monitors, or possibly even just one.

Hope this is the right place to submit a feature request :)

Thanks!

**Comment by marticliment** [comment]
Yes, i could add some kind of monitor blacklisting

**Comment by marticliment** [comment]
This feature is coming with the next version. stay tuned!

**Reply by aydenking** [reply]
Wow, fast turnaround! Thanks a bunch!

**Reply by marticliment** [reply]
Actually, it was another discussion

**Reply by marticliment** [reply]
this is a duplicate
